tests/cases/fourslash/refactorConvertExport_namedToDefaultInModuleAugmentation1.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /foo.ts
+////export {};
+////declare module "foo" {
+////    /*a*/export function foo(): void;/*b*/
+////}
+
+// @Filename: /b.ts
+////import foo from "foo";
```

Comment:
So this import was just wrong before the refactoring?

Reply:
If so, would it be interesting to add a case where it was correct and needed updating?

Reply:
@amcasey Thanks for the review 👍🏻. I've updated the test. I noticed one tricky case, that forbids updating `import`

https://github.com/microsoft/TypeScript/blob/ed014db864165cab3f568eb8e2a0b6ea1ba5f624/src/services/refactors/convertExport.ts#L68

`exportingModuleSymbol` refers to `declare module "foo"`, because `export` is located in this module, however when `importTracker` tries to find import declarations it uses a symbol of a file that has two `declarations`

https://github.com/microsoft/TypeScript/blob/67172e41c208838da45a01513f1f183072b5efd6/src/services/importTracker.ts#L394

1. `sourceFile` (there is `export {}`)
2. `declare module "foo"`

symbol id refers to the first declaration, therefore `getSymbold(exportingModuleSymbol)` !== `getSymbold(moduleSymbol)`

https://github.com/microsoft/TypeScript/blob/67172e41c208838da45a01513f1f183072b5efd6/src/services/importTracker.ts#L204

Thus `exportingModuleSymbol` doesn't include direct imports and the import reference cannot be updated. I've changed `getDirectImportsMap` to use symbols from declarations to fix this issue. Do you think that makes sense? Or is there a better way to handle this case?

Reply:
Disclaimer: I'm not particularly familiar with this code.

I'm nervous about changing `getDirectImportsMap`, which is used for things other than this code fix.  

I think what your change does is dig down from the merged module symbol to the constituent symbols - is that right?  I'm not sure I understand why that would be appropriate, since it seems like an import of "foo" refers to both, doesn't it?  Would it make more sense to use the merged symbol at the point where the map is queried (assuming there's some way to retrieve it)?